src/validation/validator.ts

```diff
@@ -0,0 +1,43 @@
+import Ajv from 'ajv';
+import { schemas } from './json-schemas';
+
+const validator = new Ajv();
+
+for (let schemaName in schemas) {
+  addSchema(schemaName, schemas[schemaName]);
+}
+
+export function addSchema(schemaName: string, schema): void {
+  validator.addSchema(schema, schemaName);
```

Comment:
Sanity: why the one line function? For ease of unit testing?

Reply:
Ah, I see, still trying understand the design here. So why is it that we have two place where schemas are added? Is it for customization/extensibility?

Reply:
yep!